packages/dappmanager/src/domains.ts

```diff
@@ -56,10 +78,39 @@ export function getPrivateNetworkAliases(
 
 export function getExternalNetworkAlias(container: ContainerNames): string {
   const fullEns = getContainerDomain(container);
-  return `${shortUniqueDappnodeEns(fullEns)}.external`;
+  return `${getShortDnpName(fullEns)}.external`;
 }
 
 export function getPublicSubdomain(container: ContainerNames): string {
   const fullEns = getContainerDomain(container);
-  return stripBadDomainChars(shortUniqueDappnodeEns(fullEns));
+  return stripBadDomainChars(getShortDnpName(fullEns));
+}
+
+/**
+ * UTILS
+ */
+
+export function stripCharacters(s: string): string {
+  return s.replace(RegExp("_", "g"), "");
+}
+
+export function stripBadDomainChars(s: string): string {
+  // eslint-disable-next-line no-useless-escape
+  return s.replace(/[^a-zA-Z\-]+/g, "-");
+}
+
+/**
+ * - Strip container prefix
+ * - Strip .dappnode, .eth, .dnp
+ * - Strip "_"
+ *
+ * @param name "bitcoin.dnp.dappnode.eth"
+ * @returns "bitcoin"
+ * - "bitcoin.dnp.dappnode.eth" > "bitcoin"
+ * - "other.public.dappnode.eth" > "other.public"
+ */
+function getShortDnpName(dnpName: string): string {
```

Comment:
I think I do not fully understand this function. Why does it return `other.public` if dnp is `other.public.dappnode.eth`? I do not see where this is being done